src/app/login/page.tsx

```diff
@@ -39,48 +91,127 @@ const LoginPage = () => {
+      </h1>
+      <p className="mt-4">Chci se přihlásit přes:</p>
+      <ul className="my-5 flex">
+        <li>
+          <button onClick={() => window.alert('🚀 Feature coming soon! 🌟')}>
+            <FacebookIcon className="mx-3 transform transition-transform duration-200 ease-in-out hover:scale-115 hover:fill-primary" />
+          </button>
+        </li>
+        <li>
+          <button onClick={() => window.alert('🚀 Feature coming soon! 🌟')}>
+            <GoogleIcon className="mx-3 transform transition-transform duration-200 ease-in-out hover:scale-115 hover:fill-primary" />
+          </button>
+        </li>
+      </ul>
+      <hr className="w-100 border-t border-light-gray" />
+      {!errorMessage ? (
+        <p className="pt-5">Zadat přihlašovací údaje:</p>
+      ) : (
+        <p className="mt-5 text-secondary">{errorMessage}</p>
+      )}
+      <form className="mt-5 flex flex-col items-center" onSubmit={handleSubmit}>
+        <label className="w-full">
+          <span className="sr-only">Email</span>
           <input
-            className="mt-2 w-full border border-black p-2"
+            className={cx(
+              'w-full border px-5 py-3 placeholder-dark-gray focus:outline-none',
+              {
+                'border-primary':
+                  formData.email && !emailError && !errorMessage,
+                'border-secondary': emailError || errorMessage,
+                'border-medium-gray': !emailError && !formData.email,
+              },
+            )}
             type="email"
             name="email"
             value={formData.email}
             onChange={handleInputChange}
+            onBlur={validateEmail}
+            placeholder="Email"
             required
           />
+          {emailError && (
+            <p className="absolute pt-1 text-xs text-secondary">{emailError}</p>
+          )}
         </label>
-        <label className="mb-4 w-full">
-          Password:
+        <label className="mb-8 w-full">
+          <span className="sr-only">Heslo</span>
           <input
-            className="mt-2 w-full border border-black p-2"
-            type="password"
+            className={cx(
+              'mt-8 w-full border px-5 py-3 placeholder-dark-gray focus:outline-none',
+              {
+                'border-primary':
+                  formData.password && !passwordError && !errorMessage,
+                'border-red-500': passwordError || errorMessage,
+                'border-medium-gray': !emailError && !formData.email,
+              },
+            )}
+            type={showPassword ? 'text' : 'password'}
             name="password"
             value={formData.password}
             onChange={handleInputChange}
+            onBlur={() => validatePassword()}
+            placeholder="Heslo"
             required
           />
+          <button onClick={() => setShowPassword(!showPassword)} type="button">
+            <EyeIcon
```

Comment:
eyeIcon is not focusable. The user cannot navigate to it with the keyboard. As an icon, it also reads nothing